lib/tasks/taxonomy/find_nested_tags.rake

```diff
@@ -0,0 +1,55 @@
+require 'gds_api/rummager'
+require 'gds_api/content_store'
+require 'set'
+
+namespace :taxonomy do
+  desc "Find any content tagged at one taxon level, then tagged again at one of those taxon's ancestors"
+  task :find_nested_tags, [:root_taxon_base_path] => [:environment] do |_, args|
+    root_taxon_base_path = args[:root_taxon_base_path]
+
+    content_store = GdsApi::ContentStore.new(Plek.find('content-store'))
+
+    root_taxon = content_store.content_item(root_taxon_base_path).to_h
+
+    errors = find_content_tagged_at_multiple_levels(root_taxon)
```

Comment:
I wonder if there's a better name than "errors". There's nothing in the task's name or description which hints that content tagged at multiple levels is an error, and when I first saw it I assumed it was rake task error handling rather than the expected result.